src/test/groovy/pl/droidsonroids/gradle/pitest/PitestPluginTest.groovy

```diff
@@ -46,6 +47,7 @@ class PitestPluginTest extends Specification {
 
     @Issue("https://github.com/koral--/gradle-pitest-plugin/issues/116")
     void "excludes mockable Android JAR"() {
+        Assume.assumeFalse(System.getProperty("os.name", "unknown").toLowerCase(Locale.ROOT).contains("win"))
```

Comment:
Btw, [`@Ignoreif`](https://spockframework.org/spock/javadoc/2.2/spock/lang/IgnoreIf.html) could be even more readable.

(and I see that we should to add an example to javadoc for that extension :) ).

Reply:
Btw2, my old blog post - https://solidsoft.wordpress.com/2014/09/17/whats-new-in-upcoming-spock-1-0-simpler-conditional-test-execution-with-requires-and-ignoreif/

```
@IgnoreIf({ os.windows  }) 
```